awx/main/management/commands/create_oauth2_token.py

```diff
@@ -0,0 +1,26 @@
+from awx.api.serializers import OAuth2TokenSerializer
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.core.exceptions import ObjectDoesNotExist
+
+
+class Command(BaseCommand):
+    """Command that creates an OAuth2 token for a certain user. Returns the value of created token."""
+    help='Usage: awx-manage create_oauth2_token --user=username. Will return the value of created token.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--user', dest='user', type=str)
+
+    def handle(self, *args, **options):
+        if not options['user']:
+            self.stdout.write("Username not supplied. Usage: awx-manage create_oauth2_token --user=username.")
+            return
```

Comment:
Per https://docs.djangoproject.com/en/1.11/howto/custom-management-commands/#command-exceptions, should raise a `CommandError` instead, which will write the message to `stderr` and exit with a non-zero return code. This will allow anyone using the command from a script/ansible to know it failed without having to parse the `stdout`.